hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -425,7 +482,7 @@ int SaraNcpClient::disconnect() {
         return SYSTEM_ERROR_NONE;
     }
     CHECK(checkParser());
-    const int r = CHECK_PARSER(parser_.execCommand("AT+CFUN=0,0"));
+    const int r = CHECK_PARSER(parser_.execCommand(UBLOX_CFUN_TIMEOUT, "AT+CFUN=0"));
```

Comment:
Sorry, I had to remove this pattern as it wasn't working with R510.  If we want to add ncpId() checks throughout and perform different commands for R410 and R510 I guess we can do that, but the `CFUN=<fun>,<rst>` RST default is 0, so it should be the same thing, so this seemed cleaner.

Reply:
Let's add a comment somewhere perhaps if this is a known R510-specific buggy behavior, so that we don't forget later in case we add any other `CFUN` calls. There are also still cases where `x, 0` format is used, is that okay?

Reply:
I've added some comments about this by all affected commands.